doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -784,3 +785,356 @@ JCL      - d247952 based on jdk-11.0.6+6)
+SMP Wed Oct 17 15:11:25 UTC 2018 riscv64 riscv64 riscv64 GNU/Linux
+```
+
+:bulb:
+To establish another session to communicate with the target system,
+run the command via any terminal software (e.g. putty) as follows:
+```
+  ssh -p 10000 root@localhost
+  root@localhost's password: riscv
+```
+
+[5] Install all X11/development related software packages required in building the JDK.
+```
+dnf install libX11-devel libXtst-devel libXt-devel libXrender-devel libXrandr-devel libXi-devel libXext-devel
+dnf install cups-devel fontconfig-devel alsa-lib-devel freetype-devel(freetype might be skipped if already installed)
+dnf install wget git autoconf automake (mostly used in the compilation on Fedora/QEMU)
+dnf install openssl-devel (optional if the OpenSSL support is required)
+```
+
+:bulb:
+If there is anything else missing during the cross-compilation, please check the link at https://secondary.fedoraproject.org/pub/alt/risc-v/RPMS/riscv64
+to see whether it is available for downloading. Some of them might go upstream to the main repository.
+
+[6] Power off the Fedora OS as follows after all installations above are finished.
+```
+poweroff
+```
+
+[7] Mount the Fedora OS image to your system for the cross-compilation.
+```
+mkdir <your_fedora_mount_directory>
+sudo mount -o loop  stage4-disk.img  <your_fedora_mount_directory>
+
+<your_fedora_mount_directory># ls
+bin   dev  home  lib64       media  opt   root  sbin  sys  usr
+boot  etc  lib   lost+found  mnt    proc  run   srv   tmp  var
+```
+
+### 3. Compile the cross-toolchain for RISC-V
+:rocket:
+The GNU cross-toolchain project contains everything required during compilation except the OS specific headers/libraries which are already installed previously on the Fedora OS image. 
+
+[1] Run the following command to get the source of the cross-toolchain:
+```
+$ git clone --recursive https://github.com/riscv/riscv-gnu-toolchain
+```
+
+[2] When the process is complete, install all required software packages on your host system:
+```
+$ sudo apt-get install autoconf \
+                       automake \
+                       autotools-dev \
+                       curl \
+                       libmpc-dev \
+                       libmpfr-dev \
+                       libgmp-dev \
+                       gawk \
+                       build-essential \
+                       bison \
+                       flex \
+                       texinfo \
+                       gperf \
+                       libtool \
+                       patchutils \
+                       bc \
+                       zlib1g-dev \
+                       libexpat-dev
+```
+
+[3] Set up the installation path and build the cross-toolchain on your system:
+```
+cd riscv-gnu-toolchain
+./configure --prefix=/opt/riscv_toochain_linux (the path to install the cross-toolchain)
+make linux
+```
+
+:bulb:
+Please refer to the latest instructions at `https://github.com/riscv/riscv-gnu-toolchain` for details if
```

Comment:
Please remove the back-ticks around the URL so it renders as a URL in a browser.

Reply:
Removed as suggested above.